Nitrox.Launcher/Models/Services/ServerService.cs

```diff
@@ -136,13 +135,24 @@ public async Task<bool> StartServerAsync(ServerEntry server)
         }
     }
 
-    public async Task<bool> ConfirmServerVersionAsync(ServerEntry server) =>
-        await dialogService.ShowAsync<DialogBoxViewModel>(model =>
+    public async Task<bool> ConfirmServerVersionAsync(ServerEntry server)
+    {
+        Version latestCompatibleVersion = new(1, 8, 0, 0); // Update this version with every release as necessary
+
+        if (server.Version >= latestCompatibleVersion)
+        {
+            return true;
+        }
+
+        // TODO: Add prompt for upgrade here if the version is incompatible but can be upgraded via upgrade script
+
```

Comment:
This doesn't seem right to me

I would rather limit the scope to major versions for now rather than hardcoding, and add a proper logic that keep tracks of upgrades at a later date